test/controllers/util/DynamicFormUtilsSpec.scala

```diff
@@ -176,17 +177,17 @@ class DynamicFormUtilsSpec extends AnyFlatSpec with MockitoSugar with BeforeAndA
     val dateTime = LocalDateTime.now().plusDays(1)
     val rawFormToMakeRequestWith =
       ListMap(
-        "inputdate-TestProperty3-day" -> List(dateTime.getDayOfMonth.toString),
-        "inputdate-TestProperty3-month" -> List(dateTime.getMonthValue.toString),
-        "inputdate-TestProperty3-year" -> List(dateTime.getYear.toString)
+        "inputdate-FoiExemptionAsserted-day" -> List(dateTime.getDayOfMonth.toString),
```

Comment:
Is it only the FOI exemption asserted input date that has the future date validation on it?

Reply:
At the moment only _foiExemptionAsserted_ has a future date validation